mission_yaml/SEA45_M48.yml

```diff
@@ -373,11 +448,11 @@ profile_variables:
 
   instrument_ctd:
     comment:    unpumped RBR legato CTD
-    calibration_date:     "2021-03-01"
+    calibration_date:     "2020-07-23"
     calibration_report:  " "
     factory_calibrated:  "yes"
     long_name:           RBR legato seaexplorer CTD
     make_model:          RBR legato
```

Comment:
should we keep it consistent with devices file

Reply:
I'll check out what this block is doing in the processing code and try to make the formatting more like the devices file